querybook/server/models/query_review.py

```diff
@@ -0,0 +1,109 @@
+import sqlalchemy as sql
+from sqlalchemy.orm import relationship, backref
+from enum import Enum
+
+from app import db
+from const.db import now, description_length
+from lib.sqlalchemy import CRUDMixin
+from lib.utils.serialize import with_formatted_date
+
+Base = db.Base
+
+
+class QueryReviewStatus(Enum):
+    PENDING = "pending"
+    APPROVED = "approved"
+    REJECTED = "rejected"
+
+
+class QueryReview(Base, CRUDMixin):
+    __tablename__ = "query_review"
+    __table_args__ = {"mysql_engine": "InnoDB", "mysql_charset": "utf8mb4"}
+
+    id = sql.Column(sql.Integer, primary_key=True, autoincrement=True)
+    query_execution_id = sql.Column(
+        sql.Integer,
+        sql.ForeignKey("query_execution.id", ondelete="CASCADE"),
+        nullable=False,
+        unique=True,
+    )
+    query_author_id = sql.Column(
+        sql.Integer, sql.ForeignKey("user.id", ondelete="CASCADE"), nullable=False
+    )
+
+    status = sql.Column(
+        sql.Enum(QueryReviewStatus), nullable=False, default=QueryReviewStatus.PENDING
+    )
```

Comment:
for a query review, it will link to a query execution by `query_execution_id`, which also contains the review status, it seems duplicate with this review `status` to me. maybe
1) we dont really need this `QueryReviewStatus`, just use the execution status instead
2) or for the `QueryExecutionStatus`, we only introduce a general `PENDING` status, but not specific to review pending, and no rejected status.  so every execution will start with a PENDING status.